.github/workflows/release.yml

```diff
@@ -101,7 +130,14 @@ jobs:
         with:
           name: IO.app.dSYM.zip
           path: ios/IO.app.dSYM.zip
-          retention-days: 60        
+          retention-days: 60
```

Comment:
```suggestion
```
In all other cases we didn't set `retation-days`. We can default to 90 days here too. What do you think?